lib/scripts/repo_export_in_batches.rb

```diff
@@ -40,7 +40,8 @@
     )
     puts "---------\n#{converter[:model_class].name} has id: #{record_for_export.id}\n---------"
 
-    if converted_record.present?
+    converted_record_json = JSON.parse(converted_record.to_json)
```

Comment:
It's probably clear from seeing the changes, but in case it's confusing here's what was happening:

```
    converted_record = converter[:converter_class].new(
      record_for_export, collection_store.data, role_store.data
    )
```
is returning an instance of the converter class. What we really wanted to check is whether or not the resulting JSON was blank.

Reply:
I think calling `to_json` then checking if the json is empty could work. It would need a test to confirm though. 

I think your suggested changes will work too, but with some additional notes. I will reserve them until it is decided to incorporate them.